src/scripts/build_all_repos.sh

```diff
@@ -908,6 +911,23 @@ EOF
 
 }
 
+function test_rpms() {
+
+    # Not the fastest way ever
+    for norepl in $NOREPLACE_FILES; do
```

Comment:
Rather than `norepl` as a variable name, why not `file`. Would be more explicit when reading the code...